### PR TITLE
lightning layer dummy data

### PR DESCRIPTION
**Pull request by LindaAlblas**
Adds a dummy GeoJSON lightning layer (point + heatmap)

### CODE REVIEW

File: client/src/app/map/identify.service.ts [hunk 1]
```diff
@@ -70,13 +70,32 @@ export class IdentifyService {
             // So we need to wait for all the calls to finish before displaying a popup
             this.identifyAsyncCalls = {
                 pdcArcGISRest: false,
```
Comment (alastaircarson) [inline client/src/app/map/identify.service.ts]: Useful to add a comment to spell out the different layer types supported

File: client/src/app/map/identify.service.ts [hunk 2]
```diff
@@ -149,7 +176,27 @@ export class IdentifyService {
...
+     * @param map
+     * @param evt
+     */
+    private setupAndIdentifyGeoJSON(map, evt){
+        var pixel = evt.pixel;
+        var coordinate = evt.coordinate;
+        var features = [];
+        map.forEachFeatureAtPixel(pixel, function(feature, layer){
+            var layerTitle = layer.getProperties().layerTitle;
+            feature.setProperties({"dmisLayerTitle": layerTitle});
```
Comment (alastaircarson) [inline client/src/app/map/identify.service.ts]: Comment to describe why this property is added

File: client/src/app/map/identify.service.ts [hunk 3]
```diff
@@ -206,14 +253,26 @@ export class IdentifyService {
         if (result.length) {
             var tableContent = '';
             for (var i = 0, ii = result.length; i < ii; ++i) {
-                tableContent += '<h4>' + result[i].getId() + '</h4>';
+                var layerTitle = '';
+                if (result[i].getProperties().dmisLayerTitle){
```
Comment (alastaircarson) [inline client/src/app/map/identify.service.ts]: Comment to match the layer type to where the layer Title is coming from

**Comment by LindaAlblas** [comment]
@alastaircarson added requested docs

**Pull request merged**